build/tf-wrapper.sh

```diff
@@ -152,53 +265,53 @@ tf_validate() {
           popd
         fi
       fi
-      gcloud beta terraform vet "${tf_env}.json" --policy-library="${policy_file_path}" --project="${project_id}" || exit 33
+      gcloud beta terraform vet "${tf_file}.json" --policy-library="${policy_file_path}" --project="${project_id}" || exit 33
       cd "$base_dir" || exit
     else
-      echo "ERROR:  ${path} does not exist"
+      echo "ERROR: ${path} does not exist"
     fi
   fi
 }
 
+#=================================================================#
 # Runs single action for each instance of env in folder hierarchy.
+#=================================================================#
 single_action_runner() {
-  local env
-  local component
+  local leaf
+  # filter folders that does not contain Terraform configurations
   find "$base_dir" -mindepth 1 -maxdepth 1 -type d \
```

Comment:
Same comment about the variable for mindepth and maxdepth variables.

Reply:
same as the other code section